src/main/java/xyz/nkomarn/harbor/util/Config.java

```diff
@@ -5,14 +5,17 @@
 import org.jetbrains.annotations.NotNull;
 import xyz.nkomarn.harbor.Harbor;
 
+import java.util.ArrayList;
 import java.util.List;
 
 public class Config {
+    private final List<ReloadListener> reloadListeners;
```

Comment:
Is this used somewhere?

Reply:
It looks like I included some bits in the commit that were supposed to not be there from an earlier version of the code. The original idea was to have a way for code to detect and respond to a reload rather than always checking with the config file, but I ended up just always checking the File config. I have both implementations so we can go with whichever you all think is more preferable, I just committed one full and one half together by accident